src/util/schemas/MemberChangeProfileSchema.ts

```diff
@@ -21,9 +21,9 @@ export interface MemberChangeProfileSchema {
 	nick?: string;
 	bio?: string;
 	pronouns?: string;
-
-	/*
-	 * @items.type integer
+	/**
+	 * @minItems 2
+	 * @maxItems 2
 	 */
-	theme_colors?: [number, number];
+	theme_colors?: number[]; // puyo: changed from [number, number] because it breaks openapi
```

Comment:
are we not able to specify that theyre ints?

Reply:
openapi json gets generated with it as an array of objects, and trying to generate an api client from that turns it into an array of `[object Object]`. 

incorrect:
```json
{
    "items": [
        {
            "type": "number"
        },
        {
            "type": "number"
        }
    ],
	"type": "array"
}
```

correct:
```json
{
    "items": {
		"type": "number"
	},
	"type": "array"
}
```

Reply:
Hm, ok. It's just that for theme colours at least there's no validation on them being ints, since that was done by ajv

Reply:
wdym?

Reply:
I mean the schema validator would check if theyre ints so we don't do that ourselves. if you remove the int requirement from the schema then it won't be validated and we'll have to do that ourselves now